tests/runtest.py

```diff
@@ -154,6 +154,11 @@ def convert_abs_path(self, build_cmd):
         cmd[src_idx] = abs_src
         return " ".join(cmd)
 
+    def strip_tracing_flags(self, cmd):
+        cmd = cmd.replace('-pg', '').replace('-finstrument-functions', '')
+        cmd = re.sub(r'-fpatchable-function-entry=[0-9]+', '', cmd)
```

Comment:
Actually the last line belongs to the next change, but it's not a big deal and I don't mind to have it here.

Reply:
Good catch, fixed.